.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-docs/ @vtex-apps/technical-writers
-messages/ @vtex-apps/localization
+* @vtex/rpt-squad-1
```

Comment:
You have to change the team here, it must be vtex-apps/rpt-squad-1 for this organization!

Reply:
Thanks a lot for the advice!!